src/Switch-CTS_CS/main.py

```diff
@@ -335,16 +312,23 @@ def configure(self):
                             
         else:
             self._temperature_set_value = None
+
+        self.reach_humidity = self.reach_humidity == "1" or self.reach_humidity == "True" or self.reach_humidity is True
```

Comment:
I think it would make sense put the "how to interpret input values" logic into the input handling function, i.e. get_GUIparameter(), like
```
self.reach_humidity = parameter["Reach humidity"] in ["1", "True", True]
```
Then self.reach_humidity will always be a bool everywhere in the driver and does not require to think about if it was already converted or not